### DISCUSSION TITLE
Welcome to Codewars Python Solutions Discussions!

### DISCUSSION THREAD

**Discussion by hevalhazalkurt** [hevalhazalkurt/codewars_python_solutions#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DamRiz13** [comment]
Hello, I really enjoy seeing your solution, how I can do to improve my skill like yours? btw I use python language

**Reply by hevalhazalkurt** [reply]
Thank you @DamRiz13 ! I'm still trying to improve my skills too. Best way to do that solving more problems, building more apps and projects. Don't scare of problems, errors, mistakes. They will help you to be better. You can start with easy problems like Codewars - 8kyu questions or Hackerrank "30 Days of Code" set (you can also find my solutions here : https://github.com/hevalhazalkurt/Hackerrank_Python_Solutions )

**Comment by Elnur1992** [comment]
Hello, could you please write how you pushed your codewars solutions to github? What commands did you use? Including .md files, where the pattern is following:
CodeWars Python Solutions
name of a problem
Examples
Given Code
Solution

Thanks in advance!

**Reply by hevalhazalkurt** [reply]
Hi! There isn't any command, template, or pattern. All of them are made by hand.
